### DISCUSSION TITLE
Horizontal mode for the actually table.

### DISCUSSION THREAD

**Discussion by sergeushenecz** [TanStack/virtual#582] [General] [answered]
Hello everyone. I have some problem for the use it is library for the horizontal virtualized.  I can to do this but i have one problem and i don't know how it resolve. 
For example: 
If i have 365 columns with width 100px each. I  set width for the parent  36500px. All good works but my width for the cell not working and i dont see each column 100px.  

I think many people also will have this problem if used with the actual table. Maybe have around this

**Comment by piecyk** [comment]
Hi, are you setting width on each column? Overall hard to say without code sandbox example.

**Comment by sergeushenecz** [comment]
Yep. I set width for each column. But because table width 36500px and we see only for example 50 column. and 50 columns can't be width 100. because table witdth 36500 px. How i understand if table with for example 5000 px and we have 3 column we can't set each column 100px.

**Reply by piecyk** [reply, marked as answer]
Hmm, maybe this will help a bit https://codesandbox.io/s/busy-goldwasser-2493k3?file=/src/App.tsx

**Reply by sergeushenecz** [reply]
look likes good. Thanks. I got it the idea

**Reply by vansh3476** [reply]
@piecyk how can i make first and last column sticky in horizontal virtulazation using Tanstack virtual?
